api/reviewer_api/services/radactionservice.py

```diff
@@ -122,10 +134,23 @@ def deactivateredaction(
         ### Remove/Update pageflags end here ###
         return result
 
+    def __getskipdocpagesmapping(self, _documentpagesmapping, _redactionlayerid):
```

Comment:
function to get documentid and pages with redactions in it. Which can be skipped for removing the pageflag